src/Command/InfectionCommand.php

```diff
@@ -210,52 +219,99 @@ protected function configure(): void
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $adapter = $this->startUp();
+        $this->initialTest($adapter);
+        $this->infectedTest($adapter);
+        $this->check();
+
+        return 0;
+    }
+
+    /**
+     * Run configuration command if config does not exist
+     *
+     *
+     * @throws InfectionException
+     */
+    protected function initialize(InputInterface $input, OutputInterface $output): void
```

Comment:
Nothing changed for this method, but git doesn't fully understand all the movement in this file